app/views/organizations/staff/fosterers/edit.html.erb

```diff
@@ -3,4 +3,4 @@
   <% c.with_body do %>
     <%= render "form", { fosterer: @fosterer } %>
   <% end %>
-<% end %>
\ No newline at end of file
+<% end %>
```

Comment:
New line :)

Reply:
weird thing is happening for me on these lines, unless I add two empty lines, the new line doesn't show up 🤔 

![Screenshot 2024-10-18 at 11 34 28](https://github.com/user-attachments/assets/60dd2501-60cc-41c2-85c2-5143eaa0be0a)

Reply:
Weird! It's just showing one line in github so we will roll with this